tools/PI/DevHome.PI/PrimaryWindow.xaml.cs

```diff
@@ -41,7 +41,7 @@ public void ShowBarWindow()
         else
         {
             // Activate is unreliable so use SetForegroundWindow
-            PInvoke.SetForegroundWindow((HWND)DBarWindow.CurrentWindowHandle);
+            PInvoke.SetForegroundWindow(DBarWindow.CurrentHwnd);

```

Comment:
Not blocking for this PR, but we should probably have a "SetForegroundWindow" method on DBarWindow, and then we wouldn't need to expose DBarWindow's HWND(s).

Reply:
Good point. However, we will need CurrentHwnd to get the window size in snaphelper as well as need it for toolcontrol. Keeping it as is for now, will update it when we can remove those dependencies too.